src/main/python/rlbot/parsing/custom_config.py

```diff
@@ -155,15 +155,15 @@ def set_value(self, option, value, index=None):
         """
         Sets the value on the given option.
         :param option: The name of the option as it appears in the config file
-        :param value: The value that is being applied, if this section is indexed value must be a list
+        :param value: The value that is being applied. If this section is indexed then the
+        value must be a list (to be applied directly) or you must supply the index parameter,
+        which will cause the value to be inserted into an existing list.
+        :param index: If the attribute is indexed, we will use this index to insert
```

Comment:
No changes required in this PR, I just want to note that I find it a bit confusing to understand what the `is_indexed` attribute stands for upon first reading this file.

Reply:
Good point, I just pushed a small improvement to the documentation.